code/go/0chain.net/smartcontract/dbs/event/block.go

```diff
@@ -34,6 +35,11 @@ func (edb *EventDb) GetBlocksByHash(hash string) (Block, error) {
 	return block, res.Error
 }
 
+func (edb *EventDb) GetBlocksMintTotal(hashes []string) (int64, error) {
```

Comment:
Have a block range as input, and sum the amount minted in each of those blocks. Let the `to` block default to the latest block, .

Reply:
need assistance how can be a string be a range...?
eg: a number can be from 1 to 10
but for a string that is a unique id... how do we use a range?